python/common/org/python/types/ByteArray.java

```diff
@@ -776,7 +776,27 @@ public boolean __setattr_null(java.lang.String name, org.python.Object value) {
             __doc__ = "B.isdigit() -> bool\n\nReturn True if all characters in B are digits\nand there is at least one character in B, False otherwise."
     )
     public org.python.Object isdigit(java.util.List<org.python.Object> args, java.util.Map<java.lang.String, org.python.Object> kwargs, java.util.List<org.python.Object> default_args, java.util.Map<java.lang.String, org.python.Object> default_kwargs) {
```

Comment:
This is already implemented in `Bytes` class, you should reuse that method instead of rewriting it.

Reply:
Oh...should I copypasta it or call it?

Reply:
Call it.

Reply:
shall I create a utility method in Bytes class like _isalpha and change that in isalpha as done previously? just for consistency... is there any other significant reason for not doing that over there in Bytes?

Reply:
> shall I create a utility method in Bytes class like _isalpha and change that in isalpha as done previously

Sounds good.